base/unicode/utf16.jl

```diff
@@ -195,44 +195,52 @@ function convert(::Type{UTF8String}, str::UTF16String)
 end
 
 "
-Converts an already validated vector of `UInt16` or `UInt32` to a `UTF8String`
+Converts a vector of `Char` to a `UTF16String`
+
+### Returns:
+*   `::UTF16String`
+
+### Throws:
+*   `UnicodeError`
+"
+function convert(::Type{UTF16String}, chrs::Vector{Char})
```

Comment:
Same situation as all the previous PR's, this is a conversion from a vector to a string that's based on a different element type, and shouldn't be here. Neither should `convert(::Type{UTF8String}, ::Vector{Char})`.